src/account/Settings.test.tsx

```diff
@@ -302,6 +305,36 @@ describe('Account', () => {
     expect(queryByTestId('KeylessBackup')).toBeNull()
   })
 
+  it('shows keyless backup setup when flag is enabled and not already backed up', () => {
+    mocked(getKeylessBackupGate).mockReturnValue(true)
+    mocked(isKeylessBackupCompleteUtil).mockReturnValue(false)
+    const store = createMockStore()
+    const { getByTestId, getByText } = render(
+      <Provider store={store}>
+        <Settings {...getMockStackScreenProps(Screens.Settings)} />
+      </Provider>
+    )
+    expect(getByTestId('KeylessBackup')).toBeTruthy()
+    expect(getByText('setup')).toBeTruthy()
+    fireEvent.press(getByTestId('KeylessBackup'))
+    expect(navigate).toHaveBeenCalledWith(Screens.WalletSecurityPrimer)
+  })
+
+  it('shows keyless backup delete when flag is enabled and already backed up', () => {
+    mocked(getKeylessBackupGate).mockReturnValue(true)
+    mocked(isKeylessBackupCompleteUtil).mockReturnValue(true)
+    const store = createMockStore()
+    const { getByTestId, getByText } = render(
+      <Provider store={store}>
+        <Settings {...getMockStackScreenProps(Screens.Settings)} />
+      </Provider>
+    )
+    expect(getByTestId('KeylessBackup')).toBeTruthy()
+    expect(getByText('delete')).toBeTruthy()
+    fireEvent.press(getByTestId('KeylessBackup'))
+    expect(navigate).not.toHaveBeenCalled()
```

Comment:
Can we ensure that analytics get fired here?

Reply:
done